app/controllers/products_common_controller.rb

```diff
@@ -110,7 +110,7 @@ def resource_params
                                                       :auto_cancel_mins, :lock_window, :cutoff_hours,
                                                       :problems_resolvable_by_user, :restrict_holiday_access,
                                                       relay_attributes: [:ip, :ip_port, :outlet, :username, :password, :type,
-                                                                         :auto_logout, :auto_logout_minutes, :id])
+                                                                         :auto_logout, :auto_logout_minutes, :id, :mac_address, :building_room_number, :circuit_number, :ethernet_port_number ])
```

Comment:
I might break this into a new line after `:mac_address` just to keep the lines a bit shorter.  Not a big deal though.